wgpu-hal/src/gles/mod.rs

```diff
@@ -113,13 +115,16 @@ bitflags::bitflags! {
     /// change the exposed feature set.
     struct PrivateCapabilities: u32 {
         /// Support explicit layouts in shader.
-        const SHADER_BINDING_LAYOUT = 0x0001;
+        const SHADER_BINDING_LAYOUT = 1 << 0;
         /// Support extended shadow sampling instructions.
-        const SHADER_TEXTURE_SHADOW_LOD = 0x0002;
+        const SHADER_TEXTURE_SHADOW_LOD = 1 << 1;
         /// Support memory barriers.
-        const MEMORY_BARRIERS = 0x0004;
+        const MEMORY_BARRIERS = 1 << 2;
         /// Vertex buffer layouts separate from the data.
-        const VERTEX_BUFFER_LAYOUT = 0x0008;
+        const VERTEX_BUFFER_LAYOUT = 1 << 3;
+        /// Indicates that we must do a manual shader clear instead of using `gl.clear_buffer` to
+        /// workaround Mesa fastclear bugs.
+        const REQUIRES_SHADER_CLEAR = 1 << 4;
```

Comment:
```suggestion
        const DRIVER_WORKAROUND_SRGB_SHADER_CLEAR = 1 << 4;
```

open to bikesheding (@kvark)

I want to see which of these are a problem with the driver and which are a device limitation.

Reply:
A few other options: `WORKAROUND_MESA_I915_SRGB_SHADER_CLEAR`, `WORKAROUND_MESA_SRGB_SHADER_CLEAR`